ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -70,6 +70,20 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+- as: aws-ipi-proxy-whitelist-arm-f14
```

Comment:
That bug is not fixed, ideally we should hit the issue in the pj-rehearse testing, right?

Reply:
yes